### PR TITLE
Add require signature auth

### PR DESCRIPTION
**Pull request by Etienne-Carriere**
Fixes : #20

### CODE REVIEW

**Review by kvz — APPROVED** [review]
Looking good! @Acconut could you sign this off?

File: docs/resources/template.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ The following arguments are supported:
 
 - `name` - (Required) name of the Template to be added
 - `template` - (Required) JSON string of the Template's Assembly Instructions. The top-level object must be the JSON object with a `"steps"` key. For more details, see [Assembly Instructions](https://transloadit.com/docs/#assembly-instructions)
+- `require_signature_auth` - (Optional- boolean . Default : false) Use true to deny requests that do not include a signature. With Signature Authentication you can ensure no one else is sending requests on your behalf.
```
Comment (Acconut) [inline docs/resources/template.md:59]: The spacing in "Optional- boolean . Default : false" looks a bit odd for me. Maybe that can be addressed and a link to https://transloadit.com/docs/#signature-authentication as well.

**Review by Acconut — APPROVED** [review]
Only got a minor comment about the documentation. Otherwise looking good :+1:

**Comment by kvz** [comment]
Thank you @Etienne-Carriere, and @Acconut as well :pray:

**Pull request merged**